Sofa/Component/IO/Mesh/tests/VisualModelOBJExporter_test.cpp

```diff
@@ -95,6 +101,11 @@ class VisualModelOBJExporter_test : public BaseSimulationTest {
         ASSERT_NE(root.get(), nullptr) ;
         root->init(sofa::core::execparams::defaultInstance()) ;
 
+        // SimulationInitDoneEvent is used to trigger exportAtBegin
+        SimulationInitDoneEvent endInit;
+        PropagateEventVisitor pe{sofa::core::execparams::defaultInstance(), &endInit};
```

Comment:
why do you need to propagate the event yourself, it is not already done in the animationLoop ?